cmd/ocm-backplane/login/login_test.go

```diff
@@ -242,6 +242,49 @@ var _ = Describe("Login command", func() {
 			globalOpts.Manager = false
 			globalOpts.Service = false
 		})
+		It("should print a message if DisplayClusterInfo is not set to true", func() {
```

Comment:
should not print?

Reply:
maybe we can change the testing to positive test as well? so it would be should print when is set to true

Reply:
Sorry, I am not getting what needs to be changed 
`It("should print a message if DisplayClusterInfo is set to true", func() {`